docs/UserGuide.md

```diff
@@ -3,41 +3,16 @@ layout: page
 title: User Guide
 ---
 
-AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
+insurance4Insurance (i4I) is an app for insurance agents to manage clients. It helps manage client profile information for insurance agents to remember personal details about his/her client. It is optimized for use via a Command Line Interface (CLI) while still having the benefits of a Graphic User Interface (GUI). 
```

Comment:
Perhaps we should keep the product name consistent, to Insurance4Insurance, I4I

Reply:
Ok sure changed to I4I